gis/imposm/Portfile

```
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/omniscale/imposm3 0.12.0 v
+go.offline_build    no
+github.tarball_from archive
+name                imposm
+revision            0
+categories          gis
+maintainers         {@sikmir disroot.org:sikmir} openmaintainer
+license             Apache-2
+
+description         Imposm imports OpenStreetMap data into PostGIS
+long_description    {*}${description}
+homepage            https://imposm.org/
+
+checksums           rmd160  413a4278bff28af61b94d38d15ed93df5ef16a32 \
+                    sha256  ab9edc262bd79dd6ee0d5547021ecd14c9931b35abb76cdeeb7cc93433ff9e13 \
+                    size    2283404
+
+depends_lib-append  port:geos \
+                    port:leveldb
+
+set go_ldflags      "-s -w -X github.com/omniscale/imposm3.Version=${version}"
```

You can also do:

```suggestion
set go_ldflags      "-s -w -X ${go.package}.Version=${version}"
```

`go.package` is set in `go.setup`.